Packs/SentinelOne/IncidentTypes/incidenttype-SentinelOneIncident.json

```diff
@@ -6,7 +6,7 @@
     "name": "SentinelOne Incident",
     "prevName": "SentinelOne Incident",
     "color": "#295AFF",
-    "playbookId": "Mirror SentinelOne Ticket",
+    "playbookId": "playbook0",
```

Comment:
> Thank you for the changes that you have made!
> 
> Adding a few more comments:
> 
> 1. Please add documentation to the tests, in a `Given, When, Then` format.
> 2. Do not use `demisto.params()` several time in the code- instead, send it as an argument to the function and use it.

fixed the point 1. For point 2 iam using the `demisto.parms()` in three places i.e main and two are in Mirroring code, if i include that params as the argument of the mirroring function then it will take the lots of code changes i.e need to update the all commands arguments. Otherwise need to use the if-elif which is the earlier code i used at the starting of this same PR